### ISSUE TITLE
Atari CLI collects random-policy trajectories but README markets it as a benchmark

### ISSUE THREAD

**Issue by sachncs** [sachncs/tsn-affinity#14]
## Problem

`tsn_affinity/cli/atari.py:collect_atari_trajectories` (lines 59-119) collects trajectories using `env.action_space.sample()` — a uniformly random policy. These trajectories are then used to train a Decision Transformer that is supposed to model a high-quality offline RL agent. Training on random-policy data means the model learns to imitate random actions, which cannot exhibit "human-normalized" performance above 0.

## Evidence

`tsn_affinity/cli/atari.py:91-92`:

```python
for _step in range(max_steps_per_trajectory):
    action = env.action_space.sample()
    obs, reward, terminated, truncated, _ = env.step(action)
```

`tsn_affinity/cli/atari.py:31-37` hardcodes five games (`Breakout`, `Alien`, `Atlantis`, `Boxing`, `Centipede`) as "the 5 Atari games from the paper". The `README.md:23` describes the Atari CLI as a "ready-to-use" benchmark. The committed `runs/atari_real_tsn_*/results.json` files (issue #2) confirm that the run produces a uniform 1.0 performance matrix and `random_losses ≈ 3.34` — the model is barely better than uniform random.

`docs/faq.md:12-17` claims Atari is supported via "the ALE benchmark suite" implying meaningful performance, but the implementation provides no human demonstrations or pretrained-agent rollouts.

## Root Cause

The CLI was designed for end-to-end smoke testing, not for producing reproducible Atari results. The README and docs oversell what the code does.

## Impact

- Anyone running `tsn-atari --strategy tsn_affinity` expecting a paper-quality number gets meaningless output.
- The committed `runs/atari_real_*` artefacts (issue #2) advertise results that are not real Atari runs.

## Recommendation

Two paths:

1. **Honest smoke test** — rename the CLI to `tsn-atari-smoke`, document it as "random-policy trajectories only; useful for CI smoke tests, not for paper results", and drop the "5 Atari games from the paper" framing.
2. **Real benchmark** — add an `--expert-data` flag that loads a HuggingFace/LeGaris-style Atari replay dataset and switches to offline training on those trajectories. Update the docs to clarify that human-normalised scores require this option.

In either case, drop or replace the `runs/atari_real_*` artefacts.

## Acceptance Criteria

- [ ] Either: docs explicitly mark Atari CLI as smoke-only; or: a real benchmark path exists
- [ ] README no longer claims "Atari benchmark" produces paper-quality numbers without `--expert-data`

## Verification

Run `tsn-atari --help` and inspect the description; compare to `docs/faq.md` Atari section.

## Risk

Medium. Changing the public contract of the Atari CLI is a breaking change for any user who relies on the current behaviour, but at v0.3.x this is acceptable. The honest framing is far better for credibility.

**Comment by sachncs** [comment]
Fixed in commit df229c6 + README + CHANGELOG. The Atari CLI no longer markets random-policy rollouts as a benchmark. tsn-atari-collect is the dedicated random-trajectory collector; tsn-atari is the training + greedy-rollout evaluator that produces real performance numbers. The README's headline table now includes a callout explaining why tsn_affinity ACC trails tsn_core ACC on the synthetic benchmark and points to the routing-modes guide for guidance on when each strategy wins.
